public/css/main.less

```diff
@@ -48,7 +47,7 @@ input[type=checkbox]
 }
 
 body.full-screen-body-background {
-  background-color: #eeeeee;
+  background-color: @gray-lighter;
 }
```

Comment:
Why not? According to [line 15 in Bootstrap's variables.less](https://github.com/FreeCodeCamp/freecodecamp/blob/staging/public/css/lib/bootstrap/variables.less#L15), @gray-lighter is #eee, and I've been using it in all the other places where #eee is being used.